src/main/java/de/uhd/ifi/se/decision/management/jira/persistence/singlelocations/JiraIssueTextPersistenceManager.java

```diff
@@ -44,9 +44,12 @@
 public class JiraIssueTextPersistenceManager extends AbstractPersistenceManagerForSingleLocation {
 	private static final Logger LOGGER = LoggerFactory.getLogger(JiraIssueTextPersistenceManager.class);
 	private static final ActiveObjects ACTIVE_OBJECTS = ComponentGetter.getActiveObjects();
-	private static OnlineFileTrainerImpl classificationTrainer = new OnlineFileTrainerImpl();
+	private static OnlineFileTrainerImpl classificationTrainer;
 
 	public JiraIssueTextPersistenceManager(String projectKey) {
+		if (classificationTrainer == null) {
+			classificationTrainer = new OnlineFileTrainerImpl();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Possible unsafe assignment to a non-final static field in a constructor.](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5586048)